app/src/main/java/asd/org/ahorcado/models/Word.java

```diff
@@ -65,7 +65,7 @@ public void inverseLetter(){
 
     public boolean isComplete() {
         for (int i = 0; i < getWord().toCharArray().length; i++) {
-            if (getWord().toCharArray()[i] != '_') {
+            if (getWord().toCharArray()[i] == '_') {
```

Comment:
Just a note, I'm going to change this hardcoded character with a constant. Because recognize this symbol means if the letter was guessed or not.